packages/website/pages/docs/resources/taiko-glossary.md

```diff
@@ -0,0 +1,95 @@
+
+## Golden Touch Address
+
+An address with a revealed private key to transact all [anchor transactions](#Anchor-Transaction).
+
+## Incentive multipliers
+
+Time sensitive multipliers that can decrease fees and increase rewards to incentivize proposals and proofs when there are unpredictable deterrents acting against the engagement of proposers or provers.
+
+## Intrinsic Validity Function
+
+A function that is run on each proposed block to determine if the block contains valid data. If the block doesn’t contain valid data the block will be seen as a block with an empty transaction list.
+
+## Invalid Block
+
+A block that fails to pass the [Intrinsic Validity Function](#Intrinsic-Validity-Function).
+
+## Metadata (of the block)
+
+A tuple of 9 items comprising: block id, beneficiary (20-byte address), gas limit, timestamp, mixHash value, extraData value, Keccak-256 hash, the enclosing L1 block’s parent block number, the enclosing L1 block’s parent block hash.
+
+## Verified block (on-chain verified)
+
+A block is on-chain verified when its proof is submitted and its parent block is verified.
+
+## Parallel Proof
+
+Blocks can be proven in parallel because all intermediate states between blocks are known and [deterministic](#Deterministic).
+
+## Proposing a block
+
+Any willing entity can propose new Taiko blocks using the [TaikoL1 contract](#TaikoL1-Contract). Blocks are appended to a list in the order they come in (which is dictated by Ethereum L1 validators/block builders). Once the block is in the list it is [verified](#Verified-block-(on-chain verified)) and nodes can apply its state to the latest L2 state. A proposed block in Taiko is the collection of information, [Metadata](#Metadata), and a list of transactions.
+
+## RLP decodable
+
+A standard for data transfer between nodes in a space-efficient manner.
+
+## Signal Service
+
+A smart contract that can be used to exchange [cross-chain messages](<#Cross-chain-communication-(messaging)>) between L1 ↔ L2 and L2 ↔ L2 that any dapp developer can use.
+
+## Slot-availability multipliers
+
+Multipliers dependent on the number of unverified blocks aimed at offering the lowest fees and rewards when there are only a few unverified blocks and a surplus of available slots, and, on the opposite, as the number of available slots decrease, competition for the remaining slots increase for proposers, which leads to higher fees.
+
+## TaikoL1 Contract
+
+The smart contract on Ethereum L1 used to [propose](#Proposing-a-block), prove, and [verify](#Verified-block-(on-chain verified)) L2 blocks.
+
+## TaikoL2 Contract
```

Comment:
we should technically remove both of these. the definition of these should be here: https://taiko.xyz/docs/reference/contract-documentation/L1/TaikoL1

this means we need to:
1. add a valid natspec comment to the solidity file TaikoL1.sol and TaikoL2.sol
2. run a `pnpm export:docs` (i think) from the root of the taiko-mono repo.

we can do this together if you'd like

Reply:
if we want to leave it here that's okay too. but we should not define it here. we can at minimum say: The L1 protocol smart contract, read more here: [TaikoL1](/docs/reference/contract-documentation/etc...